runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/util/common/worker/ByteArrayShufflePosition.java

```diff
@@ -58,15 +65,15 @@ public static byte[] getPosition(@Nullable ShufflePosition shufflePosition) {
     }
     Preconditions.checkArgument(shufflePosition instanceof ByteArrayShufflePosition);
     ByteArrayShufflePosition adapter = (ByteArrayShufflePosition) shufflePosition;
-    return adapter.getPosition();
+    return adapter.getPosition().toByteArray();
```

Comment:
Can we get rid of this method since toByteArray() does a copy?

Reply:
I decided to leave this as-is, it's used in a couple places (ChunkingShuffleBatchReader) and having the `toByteArray` call is useful to avoid another null check outside at the call site.